tests/git_push_test.go

```diff
@@ -57,6 +60,13 @@ var _ = Describe("git push deis master", func() {
 
 					Specify("that user can deploy that app using a git push", func() {
 						git.Push(user, keyPath)
+						// curl the app's root URL and print just the HTTP response code
+						cmdRetryTimeout := 60
+						curlCmd := model.Cmd{CommandLineString: fmt.Sprintf(`curl -sL -w "%%{http_code}\\n" "%s" -o /dev/null`, app.URL)}
+						Eventually(cmd.Retry(curlCmd, strconv.Itoa(http.StatusOK), cmdRetryTimeout)).Should(BeTrue())
+						// verify that the response contains "Powered by" as all the example apps do
+						curlCmd = model.Cmd{CommandLineString: fmt.Sprintf(`curl -sL "%s"`, app.URL)}
+						Eventually(cmd.Retry(curlCmd, "Powered by ", cmdRetryTimeout)).Should(BeTrue())
```

Comment:
So, I think this is the only `git push` that happens anywhere in the e2e tests currently, but in the interest of keeping the tests orderly as me move forward, here's a question...

Wouldn't we _always_ want to curl after a `git push` in order to assert the success of that deployment?  If so, I think those assertions should be built right into (moved to) `git.Push(...)` in the `tests/cmd/git` package.  My intention had been that under `cmd`, we amass lots of useful bit like this that we repeats often, and that would even include assertions like these wherever possible... all with the goal of keeping the specs themselves super concise and readable.

Just food for thought.